Wikipedia/Code/ArticlePopoverViewController.swift

```diff
@@ -19,11 +19,29 @@ class ArticlePopoverViewController: UIViewController {
     
     @IBOutlet weak var articleSummaryView: UIView!
     
-    var article: WMFArticle?
+    let article: WMFArticle
+    
+    required init(_ article: WMFArticle) {
+        self.article = article
+        super.init(nibName: "ArticlePopoverViewController", bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        return nil
```

Comment:
So it's instantiated using `initWithNibName` (see line 26 above) but that doesn't seem to call `initWithCoder`. The other option would be to have `initWithCoder` look like this:
```
    required init?(coder: NSCoder) {
        self.article = WMFArticle()
        super.init(coder: coder)
    }
```
However, it seemed more appropriate to fail & return nil since it's not initializing with a valid article. Open to other suggestions though.